lib/godmin/model/scopes.rb

```diff
@@ -0,0 +1,45 @@
+module Godmin
+  module Model
+    module Scopes
+      extend ActiveSupport::Concern
+
+      module ClassMethods
+        def scope_map
+          @scope_map ||= {}
+        end
+
+        def scopifier(attr, options = {})
+          scope_map[attr] = {
+            default: false
+          }.merge(options)
+        end
+
+        def default_scopify
+          scopify = scope_map.find -> { scope_map.first } do |_key, value|
+            value[:default] == true
+          end
+
+          scopify ? scopify[0].to_s : nil
+        end
+
+        def scope_count(scope)
```

Comment:
Unused method argument - `scope`. If it's necessary, use `_` or `_scope` as an argument name to indicate that it won't be used. You can also write as `scope_count(*)` if you want the method to accept any arguments but don't care about them.